config/cable.yml

```diff
@@ -0,0 +1,9 @@
+development:
+  adapter: async
+
+test:
+  adapter: async
+
+production:
+  adapter: redis
+  url: redis://localhost:6379/1
```

Comment:
this file can be commented out or removed, as this app is not using `action_cable`. I have spun up a a new Heroku test instance to verify that commenting it out does not cause any regressions